industry_benchmarks/utils/traj_cleanup.py

```diff
@@ -31,12 +31,12 @@ def subsample_traj(simulation, hybrid_system_pdb, lambda_windows, outfile):
     for i in range(0, lambda_windows):
         u = mda.Universe(hybrid_system_pdb, simulation,
                          format=FEReader, state_id=i)
-        skip = round(len(u.trajectory) / 20)
+        frames = [round(i) for i in np.linspace(0, len(u.trajectory)-1, 21)]
```

Comment:
This is based on the idea that there are n picoseconds + 1 frames (because we have a zero-th frame)